types/react-native/index.d.ts

```diff
@@ -2942,7 +2942,17 @@ export interface DatePickerIOSProps extends ViewProps {
     /**
      * The currently selected date.
      */
-    date: Date;
+    date?: Date | null;
+    
```

Comment:
Looks like the CI lint task is failing here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47145/checks?check_run_id=1051165205#step:5:321

Reply:
thanks! removed.